src/Exceptions/HttpTooManyRequests.php

```diff
@@ -8,4 +8,3 @@ class HttpTooManyRequests extends HttpException {
     public const DESCRIPTION = "Indicates that the user has sent too many requests in a given amount of time (rate limiting).";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces